scripts/zones/Beaucedine_Glacier/IDs.lua

```diff
@@ -29,6 +29,7 @@ zones[xi.zone.BEAUCEDINE_GLACIER] =
         YOU_CANNOT_ENTER_DYNAMIS       = 7882,  -- You cannot enter Dynamis - [Dummy/San d'Oria/Bastok/Windurst/Jeuno/Beaucedine/Xarcabard/Valkurm/Buburimu/Qufim/Tavnazia] for <number> [day/days] (Vana'diel time).
         PLAYERS_HAVE_NOT_REACHED_LEVEL = 7884,  -- Players who have not reached level <number> are prohibited from entering Dynamis.
         DYNA_NPC_DEFAULT_MESSAGE       = 7894,  -- There is an unusual arrangement of branches here.
+        GARRISON                       = 8244,  -- Hm? What is this? How do I know this is not some trick?.
```

Comment:
This could do with a better name, also, `?.` looks kind of fishy, my extracts list it as:
```
Hm? What is this? %? How do I know this is not some [San d'Orian/Bastokan/Windurstian] trick?
```

(we need this to line up with the extract, for our update tooling to continue to work)

https://raw.githubusercontent.com/zach2good/UpdateExtractor/versions/2021-december/out/scripts/zones/Beaucedine_Glacier/Text.lua.raw

Reply:
Also `GARRISON_BASE` would be an OK name